### PR TITLE
Test and document usage of BP_NPM_START_SCRIPT

### PR DESCRIPTION
**Pull request by c0d1ngm0nk3y**
## Summary

`BP_NPM_START_SCRIPT` was already being used, but hard to discover.

## Use Cases

No new functionality.

## Checklist

* [X] I have viewed, signed, and submitted the Contributor License Agreement.
* [ ] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [X] I have added an integration test, if necessary.
* [X] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [X] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

**Comment by c0d1ngm0nk3y** [comment]
@paketo-buildpacks/nodejs-maintainers Could you have a look? No functional change, only a small change to make the feature more discoverable.

File: README.md [hunk 1]
```diff
@@ -56,6 +56,15 @@ the `BP_NODE_PROJECT_PATH` environment variable at build time either directly
 file](https://github.com/buildpacks/spec/blob/main/extensions/project-descriptor.md).
 This could be useful if your app is a part of a monorepo.
 
+## Specifying a custom start script
+
+To specify a project subdirectory to be used as the root of the app, please use
```
Comment (thitch97) [inline README.md:61]: I don't think this section matches its title, seems copied from above.

Comment (c0d1ngm0nk3y) [reply README.md]: > I don't think this section matches its title, seems copied from above.

True. There was something missing. I updated the PR. Thanks.

**Review by thitch97 — DISMISSED** [review]

File: integration/testdata/app_with_start_cmd/package.json [hunk 2]
```diff
@@ -16,7 +16,8 @@
   "scripts": {
     "poststart": "echo \"poststart\"",
     "prestart": "echo \"prestart\"",
-    "start": "echo \"start\" && node server.js",
+    "start": "echo \"start\" && node hello`.js",
```
Comment (thitch97) [inline integration/testdata/app_with_start_cmd/package.json:19]: @c0d1ngm0nk3y Looks like there's an extra backtick here that's causing the tests to fail

**Comment by thitch97** [comment]
Happy to merge this once the integration tests are passing!

**Comment by c0d1ngm0nk3y** [comment]
> Happy to merge this once the integration tests are passing!

Sorry for the delay. Was on vacation. I have fixed the testdata and rebased the pr. Locally, the tests are fine now.

**Comment by c0d1ngm0nk3y** [comment]
> Happy to merge this once the integration tests are passing!

All tests seem to pass now :)

**Comment by c0d1ngm0nk3y** [comment]
@paketo-buildpacks/nodejs-maintainers Any chance of getting this merged? All tests look fine and it is basically only missing documentation/test of an existing feature.

**Review by thitch97 — APPROVED** [review]

**Pull request merged**